app/src/main/java/cm/aptoide/pt/AptoideApplication.java

```diff
@@ -345,25 +346,35 @@ public void initializeMoPub() {
         BuildConfig.MOPUB_BANNER_50_HOME_PLACEMENT_ID).withAdditionalNetwork(
         AppLovinBaseAdapterConfiguration.class.toString())
         .withMediatedNetworkConfiguration(AppLovinBaseAdapterConfiguration.class.toString(),
-            getMediationNetworkConfiguration(BuildConfig.MOPUB_BANNER_50_HOME_PLACEMENT_ID))
+            getPlacementMediationMap(BuildConfig.MOPUB_BANNER_50_HOME_PLACEMENT_ID))
```

Comment:
Personally i would call it getMediatedNetworkConfigurationBaseMap for the placement one, and then i would call getMediatedNetworkConfigurationWithAppIdMap for the app id one.